tyk-headless/templates/service-pmp.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.pump.service -}}
+apiVersion: v1
+kind: Service
+metadata:
+  name: pump-svc-{{ include "tyk-headless.fullname" . }}
+  labels:
+    app: pump-svc-{{ include "tyk-headless.fullname" . }}
+    chart: {{ include "tyk-headless.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  ports:
+  - port: {{ .Values.pump.service.port }}
+    targetPort: {{ .Values.pump.containerPort }}
+    protocol: TCP
+    name: http
```

Comment:
I was thinking of leaving it in case someone might want to modify the service to add more ports?

Reply:
Ah okay, but presently for this service we will be opening only one port through ENV var for prom pump. So, i guess we can later add this as feature in case if pump needed to expose multiple ports.

Reply:
Done